config-templates/salesforce/opportunities.yaml

```diff
@@ -0,0 +1,117 @@
+data_source:
+  name: salesforce_oppportunities
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.opportunity ### Update to your opportunities schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 10 * * *
+  identifiers:
+    - name: opportunity_id
```

Comment:
Do you like using the full id name or just the entity name here? I have a slight preference for just using the entity name since the dimensions come out cleaner: `opportunity__lead_source` vs `opportunity_id__lead_source`

Reply:
yea that makes sense, i like this format as well. I'll update this.